specs/pvs/corla.pvs

```diff
@@ -422,6 +455,7 @@ BEGIN
   original_ballot: TYPE FROM ballot
```

Comment:
All of the different flavors of ballot mentioned in the literature are all subtypes of `ballot`.  E.g., `digital_ballot` and `paper_ballot`.  The current specification does not denote if any of those subtypes are identical to, or disjoint from, any other.  If/when CDOS or CO law dictates such relationships, we will encode them in the specification and the software.

A bit of background for clarity: PVS's notion of type and subtype are using the types-as-sets interpretation.  So for each type you see declared, you should think of it as a set with a classifier (in the standard set theoretic sense).  Thus a subtype is nothing more than a subset classifier.